crates/testing/src/modules.rs

```diff
@@ -223,3 +226,43 @@ pub struct LoggerRecord {
     pub line_number: Option<u32>,
     pub message: String,
 }
+
+const COMPILATION_MODE: CompilationMode = if cfg!(debug_assertions) {
```

Comment:
Do we know if the `debug_assertions` feature is currently enabled in the transitive dependencies of the `bench` crate?

Reply:
No, but it doesn't have to be - the point here is to use that mode only in integration test mode for easier debugging (it helped me with a stacktrace).